drivers/video/video_mcux_mipi_csi2rx.c

```diff
@@ -48,6 +49,25 @@ const struct mipi_csi2rx_tHsSettleEscClk_config tHsSettleEscClk_configs[] = {
 	{MHZ(96), 0x09},
 };
 
+static int32_t get_pixel_rate_from_sensor(const struct device *sensor)
+{
+	struct video_device *sensor_vdev = video_device_find(sensor);
+
+	if (!sensor_vdev) {
+		LOG_ERR("Sensor device is not registered");
+		return -ENODEV;
+	}
+
+	struct video_ctrl *ctrl = video_ctrl_find(sensor_vdev->ctrl_handler, VIDEO_CID_PIXEL_RATE);
```

Comment:
Is `video_ctrl_find()` to use instead of `video_get_ctrl()` the way forward? Or maybe it was just for code compactness/clarity.